tests/python/gaia-ui-tests/gaiatest/apps/bzlite/app.py

```diff
@@ -0,0 +1,48 @@
+from marionette import Marionette
+from gaiatest.apps.base import Base
+from marionette_driver import By, Wait, expected
+
+class BugzillaLite(Base):
+
+    name = 'Bugzilla Lite'
+    _given_username = (By.CSS_SELECTOR, "#login input[type='email']")
+    _given_password = (By.CSS_SELECTOR, "#login input[type='password']")
+    _button_login = (By.CSS_SELECTOR, "#login input[type='submit']")
+    _profile_icon = (By.CSS_SELECTOR, ".profile")
```

Comment:
Like for the `_button_logout`, we can use the `href` attribute of the `<a>` element. So, the selector won't tap another attribute if one day we add another element with the `profile` class. I leave you the pleasure to find how this `href` locator will look like ;)